pkg/asset/installconfig/powervs/client.go

```diff
@@ -516,3 +517,23 @@ func (c *Client) GetAuthenticatorAPIKeyDetails(ctx context.Context) (*iamidentit
 func (c *Client) GetAPIKey() string {
 	return c.APIKey
 }
+
+// GetVPCs gets all VPCs in a region
+func (c *Client) GetVPCs(ctx context.Context, region string) ([]vpcv1.VPC, error) {
+	ctx, cancel := context.WithTimeout(ctx, 1*time.Minute)
+	defer cancel()
+
+	err := c.SetVPCServiceURLForRegion(ctx, region)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to set vpc api service url")
+	}
+
+	vpcs, _, err := c.vpcAPI.ListVpcs(c.vpcAPI.NewListVpcsOptions())
+	if err != nil {
+		return nil, err
+	}
+
+	allVPCs := []vpcv1.VPC{}
+	allVPCs = append(allVPCs, vpcs.Vpcs...)
+	return allVPCs, nil
```

Comment:
```suggestion
	return vpcs.Vpcs, nil
```
Any reason to create a new slice instead of returning the existing one?

Reply:
Will be addressed in the next force-push.